docs/en/integrations/language-clients/rust.md

```diff
@@ -332,6 +332,13 @@ This example relies on the legacy Hyper API and is a subject to change in the fu
 See also: [custom HTTP client example](https://github.com/ClickHouse/clickhouse-rs/blob/main/examples/custom_http_client.rs) in the client repo.
 
 ## Data Types
+
+:::info
+See also the additional examples:
+* [Simpler ClickHouse data types](https://github.com/ClickHouse/clickhouse-rs/blob/main/examples/data_types_derive_simple.rs)
+* [Container-like ClickHouse data types](https://github.com/ClickHouse/clickhouse-rs/blob/main/examples/data_types_derive_containers.rs)
```

Comment:
this file doesn't exist in main?

Reply:
Yes, as it is from https://github.com/ClickHouse/clickhouse-rs/pull/147

> To be merged after https://github.com/ClickHouse/clickhouse-rs/pull/147

It should be there afterward.